Test/Calculator.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from '../src/Components/Calculator';
+
+describe('Calculator component', () => {
+  it('renders correctly', () => {
+    const component = renderer.create(<Calculator />);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Great work using react-test-renderer to test the `Calculator` component. Based on the requirements, you ought to test all components. Kindly use it to test all major components (`Home` and `Quotes` components) created in this application. 👍 😎

Reply:
This change has been done well